README.md

```diff
@@ -238,6 +240,21 @@ Defines what meta fields content element has.
     }]
   ```
 
+### `WORKFLOWS`
```

Comment:
We are missing some details here

Reply:
Updated with more details on workflow and it's options.

Reply:
@underscope would it be helpful to link workflow section with tailor config and schema sections to make it easy to jump back and forth?